microfonia/static/microfonia/template-mobile.css

```diff
@@ -0,0 +1,74 @@
+@media only screen and (max-width: 1024px) {
```

Comment:
Na verdade eu to comentando aqui pq não consegui comentar acima, mas que que vc fez nesse jquery?